### DISCUSSION TITLE
VTM Maintenance / Future Plans

### DISCUSSION THREAD

**Discussion by kancko** [mapsforge/vtm#1135] [Q&A] [answered]
Hello there,

I noticed that the repo activity isn't high last time, so I wanted to ask if you are going to continue support and development of this project?

I would greatly appreciate it if you have any roadmap that you can share

**Comment by devemux86** [comment, marked as answer]
@kancko 

There is activity when there is something to implement or improve.
It is after all an open source library, so everyone can contribute. :slightly_smiling_face: 

Like the new Android 16 KB page size requirement:
- https://github.com/mapsforge/vtm/issues/1125

**Reply by kancko** [reply]
@devemux86 thanks for quick response
I'm just curious about roadmap/release plans cause this lib looks good to me and rendering performance quite impressing comparing to OsmDroid

So, just wanna know if you has any future plans for this lib )
